geomesa-jupyter/README.md

```diff
@@ -0,0 +1,46 @@
+* A sample notebook session
+
+```scala
+   classpath.addRepository("http://download.osgeo.org/webdav/geotools")
+   classpath.addRepository("http://central.maven.org/maven2")
+   classpath.addRepository("https://repo.locationtech.org/content/repositories/geomesa-releases")
+   classpath.addRepository("file:///home/username/.m2/repository")
+   classpath.add("com.vividsolutions" % "jts" % "1.13")
+   classpath.add("org.locationtech.geomesa" % "geomesa-accumulo-datastore" % "1.2.6")
```

Comment:
should update these to 1.3 something... I guess 1.3.0 even though it hasn't been released. Ditto 2 lines down